android/src/main/java/com/rnziparchive/RNZipArchivePackage.java

```diff
@@ -18,6 +18,11 @@
   }
 
   @Override
+  public List<Class<? extends JavaScriptModule>> createJSModules() {
+    return Collections.emptyList();
+  }
+
```

Comment:
Sorry for didn't see these lines.

They were actually deleted from this PR https://github.com/mockingbot/react-native-zip-archive/pull/48 by @javache, from the react-native team.

As he says, its no longer needed for newer version of rn, so maybe we need deletes again and marks the next release as a breaking change.

@benvium  cc @javache